src/os/im/action/filteractionentry.js

```diff
@@ -14,6 +14,7 @@ const IImportAction = goog.requireType('os.im.action.IImportAction');
  * Filter entry that performs actions on matched data.
  *
  * @implements {IComparable<FilterActionEntry>}
+ * @unrestricted
```

Comment:
We typically add this for bracket access, is that being used on this class somewhere?